geps/gep-1709.md

```diff
@@ -280,6 +318,7 @@ implementation:
   - @acme/maintainers
 date: "2023-02-28 20:29:41+00:00"
 gatewayAPIVersion: v0.7.0
+gatewayAPIChannel: standard
```

Comment:
Does this mean that most implementations would need to submit separate reports for each Gateway API Channel? I'm not sure we gain any value from that since the experimental channel is always a superset of the standard channel, but I may be missing something here.

I agree that the version of tests that are being run is critical though, so I'm glad that's included here.

Reply:
Since the experimental channel is a superset of the standard one, we could treat the report in the same way, which means being conformant to the experimental channel implies being conformant to the standard one. So that you just need to submit the experimental report to declare the standard conformance as well. WDYT?

Reply:
That's kinda what I'm thinking - we may just want to recommend that implementations that support experimental features run conformance tests with experimental CRDs. I'm not sure that we need to actually state that in the report, but don't feel strongly either way.